src/client/components/OIbForm/OIbScanMode.jsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { useParams } from 'react-router-dom'
 import { FormGroup, FormText, Label, Input, FormFeedback } from 'reactstrap'
 import { ConfigContext } from '../../context/configContext.jsx'
 
-const OIbScanMode = ({ label, help, valid, value, name, onChange }) => {
+const OIbScanMode = ({ label, help, valid, value, name, onChange, point }) => {
   const { newConfig } = React.useContext(ConfigContext)
   const scanModes = newConfig?.engine?.scanModes ?? [] // scan modes defined in engine
-  const options = scanModes.map((e) => e.scanMode)
+  const { dataSourceId } = useParams()
+  const dataSource = newConfig?.south.dataSources.find((d) => d.dataSourceId === dataSourceId)
+  const scanGroups = dataSource ? dataSource[dataSource.protocol]?.scanGroups : null
+  // Scan Group protocols should only allow scan modes set in a scan group as option
+  const options = scanGroups !== undefined && point
```

Comment:
My guess is that you want to display scanGroups only in point list window. Maybe this props should have another name to better understand its role.
Please enlighten me if I misunderstand this props

Reply:
Yes, exactly this is only needed when configuring a new point. Maybe we can use another naming like: isPointConfig.

Reply:
I have removed the point flag.